pkgs/development/python-modules/sphinxcontrib-towncrier/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, sphinx
+, towncrier
+, tomli
+}:
+
+buildPythonPackage rec {
+  pname = "sphinxcontrib-towncrier";
+  version = "0.3.2a0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-fPGKycE1M//9t5TasnjGrYRfjIW/pWlC66B31UOIX6Y=";
+  };
+
+  patches = [ ./0001-Update-build-system-dependencies.patch ];
```

Comment:
Please comment about the source of the patch, whether it should me removed on the next version bump of the file, and if there is an attempt to contribute the patch upstream. Also, I personally prefer this formatting:

```nix
patches = [
  # Comment here
  ./file.patch
];
```